stable/nats/values.yaml

```diff
@@ -69,14 +69,26 @@ podLabels: {}
 ##
 # priorityClassName: ""
 
-## Update strategy, can be set to RollingUpdate or OnDelete by default.
+## nats-cluster running mode. Allowed values: statefulset (default) or deployment
+## ref:
+## - https://kubernetes.io/docs/concepts/workloads/controllers/statefulset/
+## - https://kubernetes.io/docs/concepts/workloads/controllers/deployment/
+mode: "statefulset"
```

Comment:
We could add some validation in the NOTES.txt to check the user is not using an invalid mode (only "statefulset" and "deployment" should be allowed)

Reply:
will do, thanks